
builtin_interfaces/Time stamp
float32 setpoint          # setpoint in radiands

# TOPICS offboard/controller_input_setpoint 
